### PR TITLE
updating https://docs.datacommons.org/contributing/

### PR DESCRIPTION
**Pull request by beets**
Adding edits to the contributions page. I'm thinking now that we should leave documentation related to adding data to the data repo (we have a docs folder there, and could migrate [this page](https://github.com/datacommonsorg/docsite/blob/master/contributing/adding_datasets.md) to replace the older version [here](https://github.com/datacommonsorg/data/tree/master/docs))

### CODE REVIEW

File: contributing/index.md [hunk 1]
```diff
@@ -7,14 +7,33 @@ has_children: false
...
 - [Create new curriculum](#sharing-analysis): Using Data Commons in data science and machine learning courses.
 - [Write documentation](#updating-documentation)
 
 Data Commons welcomes patches and contributions to this project from everyone, new or experienced!
 
+## Ways to contribute
+
+### Add data
+
+We welcome contributions of public data to the Data Commons Graph. We’ve bootstrapped the graph with [data](/datasets) from US Census, World Bank, CDC, NOAA, IPCC, etc. However, Data Commons is meant to be for the community, by the community. We're excited to work with you to make public data accessible to everyone. Please see [the Data Commons data repository](https://github.com/datacommonsorg/data) for more information, and a [list of imports](https://github.com/orgs/datacommonsorg/projects/10/views/1) that would be helpful to add.
```
Comment (pradh) [inline contributing/index.md:23]: It feels useful to link to both the top-level doc (for dev process) and https://github.com/datacommonsorg/data/tree/master/docs (for concepts)?

*To get started, please take a look at the resources listed [here](https://github.com/datacommonsorg/data/tree/master/docs#contributing-to-data-commons) and follow the development steps outlined [here](https://github.com/datacommonsorg/data#development).*

or something like that..

File: contributing/index.md [hunk 2]
```diff
@@ -7,14 +7,33 @@ has_children: false
...
+
+### Creating a new tool
+
+Data Commons welcomes the development of new tools that make the data on Data Commons more accessible to end users. We have an extensive [REST API](/api/rest) that can be used to power applications on top of our data. If you would like to build a new tool, contact the team at support@datacommons.org or create a PR in [the Data Commons tools repo](https://github.com/datacommonsorg/tools).
+
+### Sharing analysis
+
+Data Commons is currently posting example analyses on [the case studies page](/tutorials). To add your analysis on that page, open a PR in [the Data Commons documentation repo](https://github.com/datacommonsorg/docsite/pulls) with the title of the analysis and links to its Colab notebook and Github raw .ipynb file. Your addition to the tutorial page should look like this when raw:
```
Comment (pradh) [inline contributing/index.md:31]: Is the last sentence complete?

**Review by pradh — APPROVED** [review]
Thanks for the edits!

File: contributing/index.md [hunk 3]
```diff
@@ -7,14 +7,33 @@ has_children: false
 
 # Contribute to Data Commons!
 
-Data Commons has benefited greatly from our collaborations with different government organizations and academic institutions and are looking to expand the set of collaborative projects. In particular, we are looking for partners to:
+Data Commons has benefited greatly from our collaborations with different government organizations and academic institutions and is looking to expand the set of collaborative projects. In particular, we are looking for partners to:
 
+- [Add data](#add-data): Contribute data to the graph which will be accessible via Data Commons tools and APIs.
```
[the remark this answers is not in the document]
Comment (beets) [reply contributing/index.md:12]: sounds good - simplified this line and added info about API in the longer bit below

File: contributing/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (beets) [reply contributing/index.md:23]: thanks, done

File: contributing/index.md (hunk 2 above)
Comment (beets) [reply contributing/index.md:31]: nice catch! updated

File: contributing/index.md (hunk 1 above)
Comment (beets) [reply contributing/index.md:23]: ah, good suggestion! done

**Review by beets — COMMENTED** [review]
thanks for the review!

File: contributing/index.md [hunk 4]
```diff
@@ -20,15 +20,15 @@ Data Commons welcomes patches and contributions to this project from everyone, n
...
+We welcome contributions of public data to the Data Commons Graph. Data added will be accessible via Data Commons tools and APIs. We’ve bootstrapped the graph with these [datasets](/datasets) from US Census, World Bank, CDC, NOAA, NASA, etc. However, Data Commons is meant to be for the community, by the community. We're excited to work with you to make public data accessible to everyone. To get started, please take a look at [these resources](https://github.com/datacommonsorg/data/tree/master/docs) and follow [this development process](https://github.com/datacommonsorg/data). Here is [list of imports](https://github.com/orgs/datacommonsorg/projects/10/views/1) that would be helpful to add.
```
Comment (pradh) [inline contributing/index.md:23]: minor nit: include link to cover "these"?  [these datasets](/datasets)

Comment (pradh) [inline contributing/index.md:23]: nit: Here is *a* list of imports ...

**Review by pradh — APPROVED** [review]
Thanks for the changes!

+1 to deduping stuff across docsite and data repo.

File: contributing/index.md (hunk 4 above)
Comment (beets) [reply contributing/index.md]: nice catch, done

Comment (beets) [reply contributing/index.md]: done!

**Pull request merged**